e2e_tests/tests/fixtures/core_api/non_conforming.py

```diff
@@ -0,0 +1,23 @@
+import determined as det
```

Comment:
Please include a brief docstring at the top of the file (or a comment somewhere) explaining what it's doing.